### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by CodeRacon** [sourcegraph/cody#2949]
### Version

Version: 1.85.2 (Universal) Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681 Date: 2024-01-18T06:40:32.531Z Electron: 25.9.7 ElectronBuildId: 26354273 Chromium: 114.0.5735.289 Node.js: 18.15.0 V8: 11.4.183.29-electron.0 OS: Darwin x64 21.6.0

### Describe the bug

after updating the vsc plugin today, every time I want cody to document some code it says:

![image](https://github.com/sourcegraph/cody/assets/146719762/42bd96f5-ddda-4640-99c6-ccff724b0872)

Despite vsc is open for hours, cody seems still not having the initialization completed. 

But: Explaining, Editing or using the chat with cody in general just works fine.

### Expected behavior

I expected cody to write the code documentation (in jsdoc format) like it used to do just yesterday within seconds and very detailed too.

**Comment by kalanchan** [comment]
hey @CodeRacon could you confirm the version you're on? Cody for VS Code is currently on v1.2 and i'm unsure of how to replicate your issue

**Comment by CodeRacon** [comment]
Hey Kalan. I'm currently on v1.2.1 but also tried some previous versions: same massage appears.

Since I use the free plan, I assume cody behaves like I described because I reached the set limit of commands for this month. 

If that's the case: A hint would be nice. 

On the other hand: it is still possible to chat along with cody, despite I reached that limit too.

I hope I could help. Sorry for the late answer. Sitting in Germany, there might be some time zones between us.

Cheers 

Michael

![image](https://github.com/sourcegraph/cody/assets/146719762/3e560835-fa26-452f-a08c-c7bf43ff53f1)
